crates/bevy_render/src/mesh/mod.rs

```diff
@@ -27,5 +33,47 @@ impl Plugin for MeshPlugin {
             .register_type::<Vec<Entity>>()
             // 'Mesh' must be prepared after 'Image' as meshes rely on the morph target image being ready
             .add_plugins(RenderAssetPlugin::<Mesh, Image>::default());
+
+        let Ok(render_app) = app.get_sub_app_mut(RenderApp) else {
+            return;
+        };
+
+        render_app.init_resource::<MeshVertexBufferLayouts>();
+    }
+}
+
+#[derive(Clone, Debug)]
+pub struct MeshVertexBufferLayoutRef(pub Arc<MeshVertexBufferLayout>);
```

Comment:
Wouldn't it make sense to add a `Deref` impl as well to avoid all the `layout.0.contains` churn?

Reply:
I was worried about accidental autoderef causing `==` and `.hash()` to be inefficient. With a `Deref` impl it'd be easy to accidentally do that.

Reply:
Could you add a comment that states that so someone doesn't inadvertently add Deref to it later?

Reply:
Done